mbed/mbed.py

```diff
@@ -211,7 +227,21 @@ def pquery(command, stdin=None, **kwargs):
         else:
             raise e
 
-    stdout, _ = proc.communicate(stdin)
+    if output_callback:
+        line = ""
+        while 1:
+            s = str(proc.stderr.read(1))
+            line += s
+            if s == '\r' or s == '\n':
+                output_callback(line, s)
+                line = ""
+
+            if proc.returncode is None:
+                code = proc.poll()
+            else:
+                break
+
+    stdout, stderr = proc.communicate(stdin)
```

Comment:
I don't think we started using `stderr` here, so it may be best to revert this change.

Reply:
Reverted. Thanks